app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
```

Comment:
It would be great to include a validation for comment body presence :)

Reply:
Validation for presence is set to true.